public/pl/css/main.css

```diff
@@ -1,18 +1,21 @@
-html{box-sizing:border-box}
```

Comment:
There should not be any diffs in this compiled file (assuming no changes were made to any ``/pl/`` styling assets. Can you please check out a fresh copy of this from master/your branch or recompile it?